server/models/Profile.js

```diff
@@ -0,0 +1,41 @@
+const mongoose = require("mongoose");
+
+const reqString = {
+    type: String,
+    require: true
+};
+
+const ProfileSchema = new mongoose.Schema({
+    userId: {
+        type: mongoose.Schema.Types.ObjectId,
+        require: true,
+        ref: 'user'
+    },
+    firstName: reqString,
+    lastName: reqString,
+    email: reqString,
+    description: {
+        type: String,
+        maxlength: 300
+    },
+    gender: String,
+    birthDate: Date,
+    phoneNumber: [String],
+    hourlyRate: Number,
+    address: {
+      city: String,
+      province: String,
+      country: String,
+    },
+    isAvailable:{
+        type: Boolean,
+        default: false
+    },
+    availability: {
+        days: [String],
+        hours: String
+    },
+    profilePhoto: [String] 
```

Comment:
If this array is for storing both the profile photo and the other images that appear in the bio, I suggest that you store the profilePhoto here as a String and have another field for the bio images that stores the URL to those images as a String array. 

You may also want to set a default profile photo.